dc-cudami-editor/src/locales/en/translation.json

```diff
@@ -204,7 +204,7 @@
     "openLinkNewTab": "open in a new tab",
     "openLinkSameTab": "open in the same tab",
     "pattern": "Pattern",
-    "refId": "Reference id",
+    "refId": "Reference ID",
```

Comment:
Existiert in `dc-cudami-editor/src/locales/de/translation.json` für Deutsch schon, perfekt :+1: